docs/release-notes/2.27.0.md

```diff
@@ -0,0 +1,143 @@
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+    <th class="field-name">Date:</th>
+    <td class="field-body">April 7, 2022</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of the PMM instance. This ensures that you access the latest features of the underlying components, such as VictoriaMetrics, with all the bug fixes in place. One of the critical features of VictoriaMetrics is [stream parsing mode](https://docs.victoriametrics.com/vmagent.html#stream-parsing-mode), which enhances the performance of PMM and saves memory when scraping targets expose millions of metrics.
+
+## Release Highlights
+
+### PMM and Percona Portal Integration
+
+-  **Enhanced UX for connecting PMM server to Percona Portal**: To leverage Percona Platform's recent support for federated identity, PMM now uses access-token authorization for connecting PMM instances to Percona Platform. 
+
+    This replaces the former username/password authentication used in PMM 2.26 and older versions.
+
+ - **Access to account information for Percona customers**: Percona customers who connect their PMM instances to the Percona Platform will now be able to see their account Entitlements and Support tickets.
+
+### Advisors
+- **Broader database health assessments with Advisors**: PMM 2.27 introduces new sets of checks to cover additional database health assessments. All checks are grouped into sets of Advisors, according to the functionality and recommendations they provide.
+
+- **New Paid tier and special Advisor checks for customers**: Customers who connect their PMM instance to Percona Platform now also have access to paid Advisor checks, which offer more advanced database health information.
+- Advisor checks have been extended with two new query types: *GetDiagnosticData* and *replSetGetStatus*. 
+
+- Improved the documentation around [developing checks](https://github.com/percona/pmm-doc/blob/NewMongoDBChecks/docs/details/develop-checks.md).
+
+### Components upgrade
+  - Grafana: PMM 2.27.0 has now migrated to Grafana 8.3.5. This version of Grafana is loaded with a gamut of exciting features. For more information, see [What's new in Grafana v8.0](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v8-0/).
+  - VictoriaMetrics: VictoriaMetrics has been upgraded to 1.72.0.
+
+
+## New Features
+
+- [PMM-9718](https://jira.percona.com/browse/PMM-9718): Federated connections to Percona Platform.
```

Comment:
This is something like "changes  to PMM to support SocialLogin  for Percona account and ability to connect PMM to the portal."

Need to be at the bottom of the section. Probably moved to Improvements as the "connect to portal" feature already exists.